src/main/resources/com/hcl/appscan/jenkins/plugin/Messages.properties

```diff
@@ -56,7 +56,8 @@ error.invalid.scan.id.rescan.allowed.ui = Rescan disabled by admin policy.
 error.base.scan.empty.ui = Invalid Scan ID. Please enter a valid Scan ID to select a base scan.
 error.token.authentication = Could not validate your Key ID and Secret. Please check and try again.
 error.scan.status.unstable= One scan execution failed. Refer to the build summary page for the results of the successful scan.
-
+error.file.type.invalid.json= Invalid file type. Only .json file is supported.
```

Comment:
The 2 message strings should be combined into a single message that takes the type as a paramter. For example:
`error.file.type.invalid=Invalid file type. Only {0} files are suported.`